assets/css/main.css

```diff
@@ -325,10 +334,38 @@ html, body {
     border-color: #bce8f1;
 }
 
+@media (min-width: 1200px) and (max-width: 1800px) {
+    .frontpage .card {
+        width: 23%;
+    }
+}
+
+@media (min-width: 960px) and (max-width: 1200px) {
+    .frontpage .card {
+        width: 30%;
+    }
+}
+
+@media (min-width: 670px) and (max-width: 960px) {
+    .frontpage .card {
+        width: 47%;
+    }
+}
+
+@media (max-width: 670px) {
+    .frontpage .card {
+        width: 100%;
+    }
+}
+
```

Comment:
These min-widths and max-widths overlap. In border cases, it seems to use the latest applicable definition. This probably isn't a big problem but it might be better to use restrictions with a difference of a pixel (eg. `max-width: 1199px` and of another:` min-width: 1200` px (or 1200px and 1201px)).

Reply:
That is a good point! Probably won't change much, but definitely should be logical limits. Those have now been changed at: https://github.com/apluslms/a-plus/pull/608/commits/96a174516db506c94d5314fc0d29acccb0d11a0c